src/logic/map_objects/tribes/productionsite.h

```diff
@@ -481,7 +482,7 @@ class ProductionSite : public Building {
 	}
 
 protected:  // TrainingSite must have access to this stuff
-	WorkingPosition* working_positions_;
+	std::unique_ptr<std::vector<WorkingPosition>> working_positions_;
```

Comment:
Since this is initialized on building creation time and never changed this could probably just be a
```suggestion
	std::vector<WorkingPosition> working_positions_;
```